Rewrote the readme to be inline with issue: Fix readme #27 (#30)
trypear/pearai-app: README.md

# Pear: The Best AI-Powered Code Editor

> Supercharge your development in an editor designed for less coding, with AI. Forked from VSCode, PearAI aims to deliver an enhanced development experience for all.

Reviewer: ```suggestion
Speed up your development with an editor integrated with AI. PearAI is forked from VSCode, and aims to reduce the time from ideation to conception for your product development by achieving the most seamless integration with AI. By developers, for developers.
```

> This repository serves as the primary application for PearAI, with most functionalities housed within the extension/pearai submodule. We recommend focusing your work within this submodule by cloning it from https://github.com/trypear/pearai-app/.

Reviewer: ```suggestion
This repository serves as the primary application for PearAI, with most functionalities housed within the `extension/pearai` folder. We recommend focusing your work within this submodule by cloning it from https://github.com/trypear/pearai-app/.
```

We welcome contributions from the community! Whether you're fixing a bug, improving the documentation, or adding a new feature, we appreciate your help in making PearAI better. To help you get your feet wet and become familiar with our contribution process, we have a list of [good first issues](https://github.com/trypear/pearai-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) that contains things with a relatively limited scope. This is a great place to get started!

Please review our [Contributing Guide](CONTRIBUTING.md) to get started.

### Known or Common Errors
Below describes a set of known or common errors that can occur when developing with PearAI and the steps that can resolve such issues.

#### No main.js found
The following issue can occur after the build process.
```
[Error: ENOENT: no such file or directory, open '/pearai/out/vs/code/electron-main/main.js'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/code/pearai/out/vs/code/electron-main/main.js',
  phase: 'loading',
  moduleId: 'vs/code/electron-main/main',
  neededBy: [ '===anonymous1===' ]
}
```
To resolve this, follow the below steps:

>  1. Remove the build and re-ran script `rm -rf out`

Reviewer: ```suggestion
 1. Remove the build: `rm -rf out`
```

Author: I have resolved this.

>  2. Re run `./scripts/code.sh` and retry

Reviewer: ```suggestion
 2. Re-run the app: `./scripts/code.sh`
```
